sql/row.go

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sql
+
+// RowMetadata represents SQL row metadata.
+type RowMetadata interface {
+	GetColumn(index int) (ColumnMetadata, error)
+	FindColumn(columnName string) (int, error)
+	GetColumnCount() int
+	GetColumns() []ColumnMetadata
```

Comment:
is it possible to drop the `Get` prefix?

Reply:
Prefixes are dropped at https://github.com/hazelcast/hazelcast-go-client/pull/755/commits/18925866ba8c0cc4fa5ef3e0b86741a4779aaad8